### ISSUE TITLE
no "livereload" for changes of *.tns.* of sharing project

### ISSUE THREAD

**Issue by hafstad** [NativeScript/nativescript-schematics#19]
I've just tried following the installation / usage guide in the readme so I'm using the recommended version of angular cli.

After install I did the following command:
ng new --collection=@nativescript/schematics --name=my-shared-app --shared

And after that I did following inside the newly created project:
tns run ios --bundle --emulator

The app starts in the iOS simulator as expected, but I noticed that changes made in the files with the *.tns.ts are not refreshing the app. 

The console writes:

_File change detected. Starting incremental webpack compilation...
Webpack compilation complete. Watching for file changes.
Webpack build done!
main.ns.ts_

However if I change in other files such *.html the changes are triggering the app to refresh. 

Is this intended behaviour, or am I doing something wrong?

**Comment by sis0k0** [comment]
Hey @hafstad, thanks for reporting that! It's not the intended behavior for sure. We'll work on fixing it.

**Comment by iMarwan-k** [comment]
Hi, I'm having the opposite behavior. when I make a change to HTML or CSS, I don't see the change until I completely bundle the whole project. however, the change to any .ts file refresh the app and it is reflected

**Comment by jkod** [comment]
yes, also getting the same opposite behavior as @iMarwan-k - only .ts file changes casues the app to reload.
